src/data/customer/__test__/job.test.js

```diff
@@ -161,4 +168,40 @@ describe('job create', async () => {
       expect(errorResponse).toEqual(expected);
     }
   });
+
+  it('should delete the jobs', async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
+    let response;
+    try {
+      // this test is passing right now because API only considers one job Id right now although two are passed with request
```

Comment:
We need to update this test.
Also we need to test removing a job ID that's part of a different route.